examples/documentation_examples/examples.go

```diff
@@ -2097,3 +2097,200 @@ func ChangeStreamExamples(t *testing.T, db *mongo.Database) {
 
 	atomic.StoreInt32(&stop, 1)
 }
+
+// AggregationExamples contains examples of aggregation operations.
+func AggregationExamples(t *testing.T, db *mongo.Database) {
+	ctx := context.Background()
+
+	coll := db.Collection("sales")
+	coll1 := db.Collection("air_alliances")
+
+	err := coll.Drop(context.Background())
+	require.NoError(t, err)
+	err1 := coll1.Drop(context.Background())
```

Comment:
nit: Reuse the `err` variable here rather than declaring a new one. Generally, new error variables should only be declared if existing ones need to be preserved, but that isn't the case here.